tests/sensors/test_external_task.py

```diff
@@ -24,32 +23,49 @@
 from airflow.exceptions import AirflowException, AirflowSensorTimeout
 from airflow.models import DagBag, TaskInstance
 from airflow.models.dag import DAG
+from airflow.models.serialized_dag import SerializedDagModel
 from airflow.operators.bash import BashOperator
 from airflow.operators.dummy import DummyOperator
 from airflow.sensors.external_task import ExternalTaskMarker, ExternalTaskSensor
 from airflow.sensors.time_sensor import TimeSensor
 from airflow.serialization.serialized_objects import SerializedBaseOperator
 from airflow.utils.state import State
+from airflow.utils.task_group import TaskGroup
 from airflow.utils.timezone import datetime
+from tests.test_utils.db import clear_db_runs
 
 DEFAULT_DATE = datetime(2015, 1, 1)
 TEST_DAG_ID = 'unit_test_dag'
 TEST_TASK_ID = 'time_sensor_check'
+TEST_TASK_GROUP_ID = 'dummy_task_group'
 DEV_NULL = '/dev/null'
 
 
-class TestExternalTaskSensor(unittest.TestCase):
-    def setUp(self):
-        self.dagbag = DagBag(dag_folder=DEV_NULL, include_examples=True)
+class TestExternalTaskSensor:
+    def setup_method(self):
         self.args = {'owner': 'airflow', 'start_date': DEFAULT_DATE}
         self.dag = DAG(TEST_DAG_ID, default_args=self.args)
+        SerializedDagModel.write_dag(self.dag)
```

Comment:
I don't think `SerializedDagModel.write_dag` is needed if the task creates its own dagbag like it previously does.